paimon-common/src/main/java/org/apache/paimon/utils/LongCounter.java

```diff
@@ -18,42 +18,36 @@
 
 package org.apache.paimon.utils;
 
-import java.io.Serializable;
+import org.apache.paimon.data.InternalRow;
 
 /** An counter that sums up {@code long} values. */
-public class LongCounter implements Serializable {
+public class LongCounter implements SequenceNumberCounter {
```

Comment:
You don't need to change this class. This class just have 50 lines, we don't need to reuse codes for it.